docs/Examples/NYISO Capacity Prices.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
this notebook looks good as is.

if you want it to show up in the [Examples Section](https://docs.gridstatus.io/en/latest/Examples/index.html) of the documentation, add it to `Examples/index.md`. if you do want to do that, I suggest add at least one graph/visualization with plotly like the other notebooks have

Reply:
perhaps something similar to what they have in last few tabs of the excel file youre grabbing the data from

<img width="945" alt="image" src="https://user-images.githubusercontent.com/371792/199522280-362371de-7efc-4078-8af8-13b6fa7428d1.png">